docs/examples/Parameters/Parameters.ipynb

```diff
@@ -89,137 +125,118 @@
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "c is gettable True\n",
-      "c is settable True\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "print(f\"c is gettable {c.gettable}\")\n",
-    "print(f\"c is settable {c.settable}\")"
+    "\n",
+    "c = MyCounter('c')\n",
+    "\n",
+    "# c() is equivalent to c.get()\n",
+    "print('Successive calls of c.get():', c(), c(), c(), c(), c())"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Virtual Paramter with Underlying Instrument Knowledge"
+    "### Demonstration of the set method"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "First of all, it is important to understand that Virtual Parameter is not a separate class in the QCoDeS but we notice that users often create parameters over already existing parameters, we call these secondary parameters as virtual parameter. An example is as follows."
+    "# We can also set the value here\n",
+    "print ('setting counter to 22:', c(22))\n",
+    "\n",
+    "print('After, we can get', c())\n"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {
-    "ExecuteTime": {
-     "end_time": "2021-06-17T08:52:09.041533Z",
-     "start_time": "2021-06-17T08:52:09.023511Z"
-    }
-   },
-   "outputs": [],
+   "cell_type": "markdown",
+   "metadata": {},
    "source": [
-    "dac = DummyInstrument('dac', gates=['ch1', 'ch2'])"
+    "### Inspecting parameter attributes\n",
+    "\n",
+    "When developing protocols, it may be useful to inspect if a parameter is settable or gettable. This can be seen in the respective .settable and .gettable attributes produced by the `Parameter` base class.  \n"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
-   "metadata": {
-    "ExecuteTime": {
-     "end_time": "2021-06-17T08:58:47.474820Z",
-     "start_time": "2021-06-17T08:58:47.464818Z"
-    }
-   },
+   "execution_count": null,
+   "metadata": {},
    "outputs": [],
    "source": [
-    "class VirtualParameter(Parameter):\n",
-    "    def __init__(self, name, dac_param):\n",
-    "        self._dac_param = dac_param\n",
-    "        super().__init__(name)\n",
-    "        \n",
-    "    def get_raw(self):\n",
-    "        return self._dac_param.get()    "
+    "print(f\"Is c is gettable? {c.gettable}\")\n",
+    "print(f\"Is c is settable? {c.settable}\")"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 8,
-   "metadata": {
-    "ExecuteTime": {
-     "end_time": "2021-06-17T08:58:48.861043Z",
-     "start_time": "2021-06-17T08:58:48.853047Z"
-    }
-   },
-   "outputs": [],
+   "cell_type": "markdown",
+   "metadata": {},
    "source": [
-    "vp1 = VirtualParameter('param_without_underlying_instrument_knowledge', dac.ch1)"
+    "## Virtual Parameters\n",
```

Comment:
Should we replace this with or at least include delegate parameter?

Reply:
yes, i think it wouild be good too add the delegate parameter. it should also be low effort since delegateparameter docstring is relatively comprehensive so just adding it to the notebook would be enough for now